js/mixins/geometry-circles.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
This mixin is used only once, is it necessary to add complexity by putting it in a separate file? If we want to reuse it in another place in the future, we can separate it out then.

Reply:
I guess this depends on preferences, but I in my opinion it is less complex to have this logic seperated, easier to maintain, and that it makes it more likely to be reused if it is already seperated.